Kudu.Core/Infrastructure/PathUtils/PathWindowsUtility.cs

```diff
@@ -95,6 +95,12 @@ internal override string ResolveNpmJsPath()
             return Path.Combine(programFiles, "nodejs", npmCliPath);
         }
 
+        internal override string ResolveMSBuild15Dir()
+        {
+            string programFiles = SystemEnvironment.GetFolderPath(SystemEnvironment.SpecialFolder.ProgramFilesX86);
+            return Path.Combine(programFiles, "MSBuild-15.3-preview", "MSBuild", "15.0", "Bin");
```

Comment:
when I rebase, I can make it to be 2 commits, whereas one only has change related to **MSBuild15Dir**